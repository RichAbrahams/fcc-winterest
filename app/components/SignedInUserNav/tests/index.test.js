// import React from 'react';
// import { shallow } from 'enzyme';

// import SignedInUserNav from '../index';

describe('<SignedInUserNav />', () => {
  it('Expect to have unit tests specified', () => {
    expect(true).toEqual(false);
  });
});
