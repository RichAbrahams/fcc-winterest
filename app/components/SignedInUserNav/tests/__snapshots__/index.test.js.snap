exports[`test renders correctly 1`] = `
<div
  className="dqUvXD">
  <h3
    className="iaCmgk" />
  <span
    aria-hidden={true}
    className="fa fa-sign-out fa-3x frqUSI"
    onClick={[Function]} />
</div>
`;
